refurb/checks/iterable/in_tuple.py

```diff
@@ -14,7 +14,7 @@ class ErrorInfo(Error):
     Bad:
 
     ```
-    for x in [1, 2, 3]:
+    for x in (1, 2, 3):
```

Comment:
After your change this snippet is the same in both the "good" and the "bad" sections. The docstring for this check was poorly worded anyways (see #44 and #49) and will probably be re-worded in a future PR.

I would revert this one line for now (and make sure to rebuild the docs). This check will be a lot more useful when #100 is resolved, but until then the docs should probably just remain as-is.

Reply:
> After your change this snippet is the same in both the "good" and the "bad" sections.

It's not, I only changed the first iterable to show the inconsistency (that I guess this rule is supposed to point out). So it's tuple&list -> tuple&tuple 👍

Reply:
I see what you mean now. I think that the wording could be adjusted to help explain the inconsistency part a bit better (it took me a second to figure out what you meant), but perhaps that goes to show that overall this check is a bit hard to explain.